### DISCUSSION TITLE
The inference process cannot be completed successfully.

### DISCUSSION THREAD

**Discussion by HDMI-Ning** [MIC-DKFZ/nnUNet#2071] [Q&A]
Dear Fabian,
I encountered an issue while training with nnunetv1. My command was **nnUNet_train 3d_lowres nnUNetTrainerV2 Task056_VerSe all --npz**. During inference after 1000 epochs, only a portion of the predictions is completed. The inference gets stuck at the point shown in Figure 1. Figure 2 shows my validation_raw folder, where only some data can be generated properly. Additionally, the summery.json file is not generated. My computer has 128GB RAM and a GPU 3090 with 24GB
![1](https://github.com/MIC-DKFZ/nnUNet/assets/34772448/ca54e2a6-d58d-4c95-9ae4-2f9a308f5176)
. Do I need to adjust any parameters in the code to make it run properly? My data is related to the spine and includes 26 labels, which might require significant inference space.
![2](https://github.com/MIC-DKFZ/nnUNet/assets/34772448/31d28117-498e-40f1-acf7-a0b1dfa51fa1)

**Comment by Lym2333** [comment]
Hi, I'm having the same problem as you, did you fix it?
